app/models/user.rb

```diff
@@ -0,0 +1,68 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :integer          not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  remember_created_at    :datetime
+#  sign_in_count          :integer          default(0), not null
+#  current_sign_in_at     :datetime
+#  last_sign_in_at        :datetime
+#  current_sign_in_ip     :inet
+#  last_sign_in_ip        :inet
+#  confirmation_token     :string
+#  confirmed_at           :datetime
+#  confirmation_sent_at   :datetime
+#  unconfirmed_email      :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#  name                   :text             not null
+#  msf_location_id        :integer
+#  external_location      :text
+#  role                   :enum             default("normal_user"), not null
+#
+# Indexes
+#
+#  index_users_on_confirmation_token    (confirmation_token) UNIQUE
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_msf_location_id       (msf_location_id)
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#
+
+class User < ActiveRecord::Base
+  # Include devise modules. Others available are:
+  # :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable, :recoverable,
+         :rememberable, :trackable, :validatable, :confirmable
+
+  enum role: {
```

Comment:
I've fixed this up, but I might drop the commit and apply it later, as it's causing a bit of an issue with the migrations etc because of active admin being added in the next PR